tests/functional/cylc-remove/00-simple/flow.cylc

```diff
@@ -1,24 +1,26 @@
 # Abort on stall timeout unless we remove unhandled failed and waiting task.
 [scheduler]
     [[events]]
-        stall timeout = PT20S
+        stall timeout = PT30S
```

Comment:
Why was this necessary? Was it flaky without?

Reply:
Probably on MacOS GH Actions runner